lib/adapter.js

```diff
@@ -21,6 +21,7 @@ Adapter.prototype.connect = function (callback) {
 
     const self = this;
     const settings = Hoek.clone(self.params.database.connection);
+    const tableName = self.params.database.migrationTable || 'migration';
```

Comment:
>Would it make sense to handle the default in the constructor?

Yep, done in https://github.com/sonyahogan/east-rethink/pull/7/commits/73790bc26cc10abdec8e2d02fdc516f6b89e99a0